gui/src/ts/components/benchmark/BenchmarkAnswerPage.tsx

```diff
@@ -94,13 +99,18 @@ export const BenchmarkAnswerPage: React.FunctionComponent = () => {
               <Grid item container>
                 <Grid item md={6} container direction="column" justify="center">
                   <Grid item>
-                    <Typography variant="h4">{question?.text}</Typography>
+                    <Typography variant="h4" data-cy="questionText">
```

Comment:
Nit: have been doing kebab-case for these data-cy attributes, but no need to change this.